src/project-tests/survey-form-tests.js

```diff
@@ -115,7 +115,16 @@ export default function createSurveyFormTests() {
             });
 
             it('12. I can select an option from a dropdown that has corresponding id="dropdown".', function() {
-                var dropdown = document.getElementById('dropdown');
+                var dropdownid = 'dropdown';
+                var dropdown = document.getElementById(dropdownid);
+                var dropdowntag = dropdown.tagName;
+                var inputsibling = null;
+                var inputname = null;
+                if (dropdowntag === 'DATALIST') {
+                    inputsibling = dropdown.previousElementSibling;
+                    inputname = inputsibling.getAttribute('list');
+                    FCC_Global.assert.strictEqual(inputname, dropdownid, 'When using the datalist tag, the accompanying input tag must contain a name attribute matching the datalist id.');
```

Comment:
Then here, just compare `inputname` against the string `'dropdown'`.

Also, camelCasing all of these variables follows best practice and will make the code easier to read. i.e.:

`inputname` should be `inputName`
'dropdowntag` should be `dropwdownTag`
`inputsibling` should be `inputSibling` etc.

This is another case that will help you and future devs read the code a bit more easily (i'm one to talk - most of this project is pretty messy! :embarrassed:).

Great job overall on your first PR though!